packages/airnode-admin/src/sdk.ts

```diff
@@ -48,49 +52,60 @@ export class AdminSdk {
   sponsorToWithdrawalRequestCount = (sponsorAddress: string) =>
     admin.sponsorToWithdrawalRequestCount(this.airnodeRrp, sponsorAddress);
 
-  fulfillWithdrawal = (requestId: string, airnodeAddress: string, sponsorAddress: string, amount: string) =>
-    admin.fulfillWithdrawal(this.airnodeRrp, requestId, airnodeAddress, sponsorAddress, amount);
+  fulfillWithdrawal = (
+    requestId: string,
+    airnodeAddress: string,
+    sponsorAddress: string,
+    amount: string,
+    overrides?: ethers.Overrides
```

Comment:
I changed the overrides type to normal overrides instead payable since `value` is handled through the `amount` parameter.